commons/jdk21/pom.xml

```diff
@@ -17,6 +17,7 @@
       <maven.compiler.source>21</maven.compiler.source>
       <maven.compiler.target>21</maven.compiler.target>
       <maven.compiler.release>21</maven.compiler.release>
+      <maven.compiler.arguments>--enable-preview</maven.compiler.arguments>
```

Comment:
Does it also require the flag to run?

Reply:
Yeah, I forgot when I reverted the changes @tristantarrant mentioned that I need it since the new classes can only be used with this argument on 21. So I need to add back in the reflection stuff.